### ISSUE TITLE
Opening notifications may not show latest comments

### ISSUE THREAD

**Issue by vlidholt** [newsvoice/newsvoice#164]
It's possible following notifications shows cached comments and not getting the latest updates.

User feedback:
*Im getting notifications about people commenting on stories that ive commented on, but when i click the notification it doesnt show any new comments.*

**Comment by SonOfWisdom** [comment]
I agree. From notification, tapping on a story doesn't show you anything about where you commented. I'm mostly interested in my conversation.  Maybe add a way to go back to the actual story from that comment page.
